src/baseline_bots/utils.py

```diff
@@ -315,27 +315,57 @@ def get_best_orders(bot, proposal_order: dict, shared_order: dict):
     input: sender power, dipnet_order + incoming proposals {power: [orders]}, shared_orders (info about other power), Diplomacy game
```

Comment:
Why do you have input: and output: twice here?

Reply:
that's very weird. I recalled deleting the old version

Reply:
removed the old one!